File: 1-easy/problems/3467-transform-array-by-parity/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def transformArray(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        cnt = 0
        for x in nums:
            if x%2:
                continue
            nums[cnt] = 0
            cnt += 1
        for i in range(cnt, len(nums)):
            nums[i] = 1
        return nums

File: 1-easy/problems/3467-transform-array-by-parity/solution.ts
function transformArray(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: List[int]
  """
  cnt = 0
  for (x in nums) {
      if (x%2) {
          continue
      nums[cnt] = 0
      cnt += 1
  for (i in Array.from({length: cnt, (nums}, (_, i) => i).length)) {
      nums[i] = 1
  return nums;
}

export { transformArray };
